lib/services/nodes-api-service.js

```diff
@@ -30,64 +32,161 @@ function nodeApiServiceFactory(
     }
 
     /**
-     * Find Enclosure nodes which enclose compute node
+     * Find nodes downstream that relate with the removing node
      * @param  {Object}     node
-     * @return {Promise}
+     * @param  {Object}     type
+     * @return {Promise}    array of downstream nodes
      */
-    NodeApiService.prototype._findEnclNodes = function(node) {
+    NodeApiService.prototype._findDsNodes = function(node, type) {
```

Comment:
DsNodes = downstream nodes, I stated the latter in PR description and the doc for this function.